GitUI/Script/ScriptOptionsParser.cs

```diff
@@ -109,20 +117,20 @@ public static (string argument, bool abort) Parse(string argument, IGitModule mo
                         }
                     }
                 }
-                else if (option.StartsWith("s") && selectedRevision == null && revisionGrid != null)
+                else if (DependsOnSelectedRevision(option) && selectedRevision == null && revisionGrid != null)
```

Comment:
Please perform `null` checks first

Reply:
I can change this. I have just factored out the existing quite simple an IMO cheap condition `.StartsWith`.

Do you suggest this for performance reasons? In general, for understandability of the code, I prefer a logical sequence of conditions over a performance-optimized. Else (not in this case) comments could be needed, which tend to be not maintained.

Then I would perform the `null` check here first, too:
https://github.com/gitextensions/gitextensions/blob/7fb79b9aab842d6635e5908c3cdea27a1437dd38/GitUI/Script/ScriptOptionsParser.cs#L102

Reply:
In cases like this I favour perf - it doesn't really impede readability but guards against possible perf implications. 
null checks are cheap and a function call may be quite expensive (load it into the stack, and execute). It is not impossible that a method small and benign today may turn into a multiline monstrosity tomorrow.

Reply:
done